interp/src/interpreter/interpret_control.rs

```diff
@@ -99,19 +70,9 @@ fn eval_if(
     )?;
 
     if cond_flag == 0 {
-        env = interpret_control_inner(
-            &i.fbranch,
-            continuous_assignments,
-            env,
-            comp,
-        )?;
+        env = interpret_control(&i.fbranch, continuous_assignments, env, comp)?;
```

Comment:
This can be turned into a simpler if expression since the only difference is choosing which branch to execute based on the condition.